platform/linux-generic/odp_pool.c

```diff
@@ -853,7 +857,8 @@ int odp_pool_capability(odp_pool_capability_t *capa)
 	capa->pkt.max_pools        = ODP_CONFIG_POOLS;
 	capa->pkt.max_len          = CONFIG_PACKET_MAX_SEGS * max_seg_len;
 	capa->pkt.max_num	   = CONFIG_POOL_MAX_NUM;
-	capa->pkt.min_headroom     = CONFIG_PACKET_HEADROOM;
+	capa->pkt.min_headroom     = 0;
```

Comment:
Actually, min_headroom is still CONFIG_PACKET_HEADROOM since we always round up to that.

Reply:
Given Petri's other comments, I agree here. Sounds like we just want to make these additional params available in the spec but ignore them in odp-linux. We can revisit this in the cloud-dev branch.